package.json

```diff
@@ -135,17 +135,14 @@
     "@vue-macros/common": "3.0.0-beta.8",
     "ast-walker-scope": "^0.7.1",
     "chokidar": "^4.0.3",
-    "fast-glob": "^3.3.3",
     "json5": "^2.2.3",
     "local-pkg": "^1.1.1",
     "magic-string": "^0.30.17",
     "mlly": "^1.7.4",
     "pathe": "^2.0.3",
     "picomatch": "^4.0.2",
     "scule": "^1.3.0",
-    "unplugin": "^2.3.2",
-    "unplugin-utils": "^0.2.4",
-    "yaml": "^2.7.1"
```

Comment:
why are these removed though?

Reply:
Not sure what happened there. Thanks for fixing it for me!

Reply:
No worries!